test/junit/scala/concurrent/duration/DurationTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.concurrent.duration
```

Comment:
FWIW duration-tck is also pure standard library testing. I'd rather keep related tests together so that migration (if needed) can be done wholesale.